apps/functions-subscription/src/domain/__tests__/trial.test.ts

```diff
@@ -23,6 +24,30 @@ describe('insertTrial', () => {
     expect(testEnv.trialWriter.insert).toBeCalledWith(aTrial);
   });
 
+  it('should return all trials', async () => {
+    const testEnv = makeTestEnv();
+
+    const anotherTrial = {
+      ...aTrial,
+      id: 'anotherTrialId012345678901' as TrialId,
+      name: 'anotherTrialName' as NonEmptyString,
+      description: 'anotherTrialDescription',
```

Comment:
**nitpick**: you can omit to update every parameter

Reply:
Removed description in eff99bbaf41355bc1a98ee131ab9a4d2694136f4